src/app/personside/visittkort-v2/body/kontaktinformasjon/adresse/Adresse.tsx

```diff
@@ -5,33 +5,48 @@ import { Normaltekst } from 'nav-frontend-typografi';
 import { Adresse as PersonAdresse, Person } from '../../../PersondataDomain';
 import Adresseinfo from '../../AdresseInfo';
 import Endringstekst from '../../Endringstekst';
+import GyldighetsPeriode from '../../GyldighetsPeriode';
 
 interface Props {
     person: Person;
 }
 
-function AdresseElement(props: { adresse: PersonAdresse | null; beskrivelse: string }) {
-    if (!props.adresse) {
+interface AdresseElementProps {
+    adresse: PersonAdresse | null;
+    beskrivelse: string;
+    erOppholdsadresse?: boolean;
+}
+
+function AdresseElement({ adresse, beskrivelse, erOppholdsadresse }: AdresseElementProps) {
+    if (!adresse) {
         return (
-            <VisittkortElement beskrivelse={props.beskrivelse} ikon={<LocationPin />}>
+            <VisittkortElement beskrivelse={beskrivelse} ikon={<LocationPin />}>
                 <Normaltekst>Ikke registrert</Normaltekst>
             </VisittkortElement>
         );
     }
 
+    const skalViseGyldighetsPeriode =
+        erOppholdsadresse || (!erOppholdsadresse && adresse.gyldighetsPeriode?.gyldigTilOgMed);
+
     return (
-        <VisittkortElement beskrivelse={props.beskrivelse} ikon={<LocationPin />}>
-            <Adresseinfo adresse={props.adresse} />
-            <Endringstekst sistEndret={props.adresse.sistEndret} />
+        <VisittkortElement beskrivelse={beskrivelse} ikon={<LocationPin />}>
+            {skalViseGyldighetsPeriode && <GyldighetsPeriode gyldighetsPeriode={adresse.gyldighetsPeriode} />}
```

Comment:
Gjør det noe om vi fjerner denne sjekken? Så blir alle adressene på samme format?

Reply:
Visstnok var det et ønske at gyldighetsperiode alltid vises for Oppholdsadresse, men at Bostedadresse og Kontaktadresse kun viser gyldighetsperiode dersom man har en gyldigTilOgMed (dersom man har gyldigTOM vil man jo også ha en gyldigFOM)